Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM golang:1.11-stretch
+# Go base image
+ARG base=golang:1.11-stretch
+# Busybox base image
+ARG busybox_base=busybox:1-glibc
```

Comment:
I'm not a docker expert, but shouldn't this just go down at line 46? Is there a reason to duplicate it?